mlflow/store/model_registry/rest_store.py

```diff
@@ -39,29 +41,67 @@
 _logger = logging.getLogger(__name__)
 
 
-class RestStore(AbstractStore):
+def _get_response_from_method(method):
+    return method.Response()
+
+
+class BaseRestStore(AbstractStore):  # pylint: disable=abstract-method
```

Comment:
Should an abstract implementation be put in a different module (just in case in the future to potentially avoid a forced refactor to eliminate a circular ref import between the rest_store and the uc_store implementations). Just a nit thought here. Feel free to disregard if it seems a silly ask.

Reply:
Seems totally reasonable & sounds good to move to a separate module :)